test/e2e/auth/node_authz.go

```diff
@@ -41,6 +42,7 @@ const (
 var _ = SIGDescribe("[Feature:NodeAuthorizer]", func() {
 
 	f := framework.NewDefaultFramework("node-authz")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelBaseline
```

Comment:
this can't be right. upstream would have hit this.

Reply:
yeah, as discussed OOB let me recheck why this passes on upstream

Reply:
I rechecked, indeed the node authorizer check is skipped upstream, at least in https://github.com/kubernetes/kubernetes/pull/106454.